easybuild/easyconfigs/q/Qt6/Qt6-6.4.2-GCCcore-12.2.0.eb

```
@@ -0,0 +1,73 @@
+easyblock = 'ConfigureMake'
```

since the build system going forward is `cmake`/`ninja`, lets use the appropriate easyblock
```suggestion
easyblock = 'CMakeNinja'
```